pangea-sdk/v3/README.md

```diff
@@ -1,60 +1,50 @@
-<p>
-  <br />
-  <a href="https://pangea.cloud?utm_source=github&utm_medium=node-sdk" target="_blank" rel="noopener noreferrer">
-    <img src="https://pangea-marketing.s3.us-west-2.amazonaws.com/pangea-color.svg" alt="Pangea Logo" height="40" />
-  </a>
-  <br />
-</p>
-
-<p>
-<br />
-
-[![documentation](https://img.shields.io/badge/documentation-pangea-blue?style=for-the-badge&labelColor=551B76)](https://pangea.cloud/docs/sdk/go/)
-[![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://pangea.cloud/join-slack/)
+<a href="https://pangea.cloud?utm_source=github&utm_medium=go-sdk" target="_blank" rel="noopener noreferrer">
+  <img src="https://pangea-marketing.s3.us-west-2.amazonaws.com/pangea-color.svg" alt="Pangea Logo" height="40" />
+</a>
 
 <br />
-</p>
+
+[![documentation](https://img.shields.io/badge/documentation-pangea-blue?style=for-the-badge&labelColor=551B76)][Documentation]
+[![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)][Slack]
 
 # Pangea Go SDK
 
-A Go SDK for integrating with Pangea Services.
+A Go SDK for integrating with Pangea services.
 
 ## Installation
 
-To add Pangea Go SDK to your project, will need to run next command on your project root directory where you should have `go.mod` file:
-
-```
-go get github.com/pangeacyber/pangea-go/pangea-sdk
+```bash
+$ go get github.com/pangeacyber/pangea-go/pangea-sdk
```

Comment:
shouldn't add `/v3`?

Reply:
So this whole time the instructions have been pointing users to v1? 😟 

Fixed.

Reply:
I'm afraid so